src/python/dxpy/scripts/dx.py

```diff
@@ -2512,23 +2512,12 @@ def install(args):
         err_exit()
 
 def uninstall(args):
-    app_desc = get_app_from_path(args.app)
-    if app_desc is None:
-        try:
-            dxpy.api.app_uninstall(args.app)
-            print('Uninstalled application with id: ' + args.app)
-        except:
-            try:
-                dxpy.api.app_uninstall('app-' + args.app)
-                print('Uninstalled application with name: ' + args.app)
-            except:
-                err_exit()
+    user_data = dxpy.api.user_describe(dxpy.whoami(), {'appsInstalled': True})
```

Comment:
Please invoke this method with request data format `{"fields": {"appsInstalled": True}}`; the previous data format has been deprecated.